pkg/version/version_test.go

```diff
@@ -0,0 +1,45 @@
+package version
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetFullVersion(t *testing.T) {
+	testCases := []struct {
+		name            string
+		expectedversion string
+	}{
+		{
```

Comment:
You didn't cover other cases. You don't need table test here.
You can test different cases in sequence like below, please refer it to add more to cover all cases.
```
	gotversion := GetFullVersion()
	assert.Equal(t, "UNKNOWN", gotversion)

	Version = "1.9"
	ReleaseStatus = "unreleased"
	gotversion = GetFullVersion()
	assert.Equal(t, "1.9-unknown", gotversion)
...
```